### PR TITLE
swi-prolog: --devel 7.1.36

### PR DESCRIPTION
**Pull request by honnix**

### CODE REVIEW

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: swi-prolog.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ class SwiProlog < Formula
   end
 
   devel do
-    url "http://www.swi-prolog.org/download/devel/src/pl-7.1.32.tar.gz"
-    sha1 "19922123f51839ae461af69bac49cdb23fbe4733"
+    url "http://www.swi-prolog.org/download/devel/src/pl-7.1.36.tar.gz"
```
Comment (DomT4) [inline swi-prolog.rb]: `7.1.37` seems to be out if you want to bump this again. Please use a `sha256` checksum if you do, thanks!

Comment (honnix) [inline swi-prolog.rb]: I think we you can close this request as I just submitted a new one for latest stable version and development version. Version 7 is finally stable!

Thanks.

Comment (DomT4) [inline swi-prolog.rb]: Great, Thanks!

**Pull request closed without merging**